web/elm/src/Views/Styles.elm

```diff
@@ -282,3 +288,97 @@ instanceGroupBadge backgroundColor =
     , style "align-items" "center"
     , style "justify-content" "center"
     ]
+
+
+commentBarWrapper : List (Html.Attribute msg)
+commentBarWrapper =
+    [ style "border" "medium dashed rgba(255, 255, 255, 0.5)"
+    , style "border-radius" "7px"
```

Comment:
Could we change these styles around a little?

1. There aren't any other cases of rounded rectangles in the UI (that I can think of), so it looks a little out-of-place - could we get rid of the `border-radius`
2. Could we also just make the border `1px solid`, rather than dashed